model/model.go

```diff
@@ -49,6 +49,9 @@ type Service struct {
 	// Runtime holds information about the programming language runtime
 	// running this service.
 	Runtime *Runtime `json:"runtime,omitempty"`
+
+	// ServiceNode holds unique information about each service node
+	ServiceNode *ServiceNode `json:"node,omitempty"`
```

Comment:
```suggestion
	Node *ServiceNode `json:"node,omitempty"`
```

As the enclosing type is called Service, it's a bit stuttery to call the field Service.ServiceNode.